core/invoker/Dockerfile

```diff
@@ -4,12 +4,14 @@
 FROM scala
 
 ENV DOCKER_VERSION 1.12.0
+ENV DOCKER_DOWNLOAD_SHA256 3dd07f65ea4a7b4c8829f311ab0213bca9ac551b5b24706f3e79a97e22097f8b
```

Comment:
May be leave comment (here or in PR) on where to get this SHA256 value. It would simplify future updates to Docker version

Reply:
@chetanmeh  got it here https://get.docker.com/builds/Linux/x86_64/docker-1.12.0.tgz.sha256
otherwise you can simply compute it locally using `shasum -a 256 docker-1.12.0.tgz` (osx) or `sha256sum docker-1.12.0.tgz` (linux)